contracts/test/ERC20toNativePointerTest.js

```diff
@@ -0,0 +1,178 @@
+
+            const tx = await pointer.transfer(recipient.evmAddress, 5);
+            await tx.wait();
+
+            expect(await pointer.balanceOf(sender.evmAddress)).to.equal(995);
+            expect(await pointer.balanceOf(recipient.evmAddress)).to.equal(5);
+
+            const cleanupTx = await pointer.connect(recipient.signer).transfer(sender.evmAddress, 5)
+            await cleanupTx.wait();
+        });
+
+        it("should fail transfer() if sender has insufficient balance", async function () {
+            let recipient = accounts[1];
+            // TODO: determine why we aren't able to extract the error message
+            await expectRevert.unspecified(pointer.transfer(recipient.evmAddress, 1001));
+        });
+    });
+
+    describe("approve()", function () {
+        it("should approve", async function () {
+            const owner = accounts[0].evmAddress;
+            const spender = accounts[1].evmAddress;
+            const tx = await pointer.approve(spender, 50);
+            await tx.wait();
+            const allowance = await pointer.allowance(owner, spender);
+            expect(Number(allowance)).to.equal(50);
+        });
+
+        it("should lower approval", async function () {
+            const owner = accounts[0].evmAddress;
+            const spender = accounts[1].evmAddress;
+            const tx = await pointer.approve(spender, 0);
+            await tx.wait();
+            const allowance = await pointer.allowance(owner, spender);
+            expect(Number(allowance)).to.equal(0);
+        });
+    });
+
+
+    describe("transferFrom()", function () {
+        it("should transferFrom", async function () {
+            const recipient = admin;
+            const owner = accounts[0];
+            const spender = accounts[1];
+            const amountToTransfer = 10;
+
+            // capture balances before
+            const recipientBalanceBefore = await pointer.balanceOf(recipient.evmAddress);
+            const ownerBalanceBefore = await pointer.balanceOf(owner.evmAddress);
+            expect(Number(ownerBalanceBefore)).to.be.greaterThanOrEqual(amountToTransfer);
+
+            // approve the amount
+            const tx = await pointer.approve(spender.evmAddress, amountToTransfer);
+            await tx.wait();
+            const allowanceBefore = await pointer.allowance(owner.evmAddress, spender.evmAddress);
+            expect(Number(allowanceBefore)).to.be.greaterThanOrEqual(amountToTransfer);
+
+            // transfer
+            const tfTx = await pointer.connect(spender.signer).transferFrom(owner.evmAddress, recipient.evmAddress, amountToTransfer);
+            const receipt = await tfTx.wait();
+
+            // capture balances after
+            const recipientBalanceAfter = await pointer.balanceOf(recipient.evmAddress);
+            const ownerBalanceAfter = await pointer.balanceOf(owner.evmAddress);
+
+            // check balance diff to ensure transfer went through
```

Comment:
```
await expect(token.transferFrom(sender, receiver, 1)).to.changeTokenBalances(
  token,
  [sender, receiver],
  [-1, 1]
);
```
^ this is apparently a thing (lets you check n balances), no need to rewrite, but I'll be trying this syntax on my next balance test